storm-shaded-deps/pom.xml

```diff
@@ -137,6 +137,22 @@
             <artifactId>netty-all</artifactId>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-handler</artifactId>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>io.netty</groupId>
+            <artifactId>netty-tcnative</artifactId>
+            <optional>true</optional>
+            <classifier>linux-x86_64</classifier>
```

Comment:
What about other archs?

Reply:
These are not needed; existing netty jars are good for mTls. I tested it locally.